parsec/core/fs/workspacefs/entry_transactions.py

```diff
@@ -97,7 +98,7 @@ async def _get_manifest_from_path(self, path: FsPath) -> LocalManifest:
     @asynccontextmanager
     async def _lock_parent_manifest_from_path(
         self, path: FsPath
-    ) -> Tuple[LocalManifest, LocalManifest]:
+    ) -> AsyncGenerator[Tuple[LocalManifest, Optional[LocalManifest]], None]:
```

Comment:
`AsyncIterator[Tuple[LocalManifest, Optional[LocalManifest]]]` to avoid the `None` send type ? (cf. https://docs.python.org/3/library/typing.html#typing.AsyncGenerator)